utils/eslint/index.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
```

Comment:
Just learning that there is a `linters` folder in the project 😮 

This packages should likely live there, right?

Later, we should look into moving all three, as well as the example project, into `/packages/` so that Greenkeeper and Lerna can manage them better...

Reply:
Yah, looks like there's a pattern for custom linters, so it would be good to follow it. I will defer to you on the packages thing. I'm still ramping on lerna.